etrago/tools/utilities.py

```diff
@@ -287,9 +288,10 @@ def buses_by_country(self, apply_on="grid_model"):
     con = self.engine
     germany_sh = gpd.read_postgis(query, con, geom_col="geometry")
 
-    path = gpd.datasets.get_path("naturalearth_lowres")
-    shapes = gpd.read_file(path)
-    shapes = shapes[shapes.name.isin([*countries])].set_index(keys="name")
+    # read Europe borders. Original data downloaded from naturalearthdata.com/
+    # under Public Domain license
+    path = Path(".") / "tools" / "shapes_europe"
```

Comment:
In general, we shouldn't add data to the github repository, esp. things like shapefiles that are not that small.   Was there no other option than adding it to the repo?

Reply:
It is also possible to download the shapes during the execution, but since the 5 files are in total 38,5 kB (and no licence is required) I think it is a good option to have it here. Also, it has the advantage that no internet connection would be needed and it would not be susceptible to changes in the webpage. If you think that it is a better idea to access the file from the webpage I could make those changes.

Reply:
The file sizes is not that big, but it adds up for each branch you have. And when there is no license needed to use it, it doesn't necessarily mean that we are allowed to republish it. So I guess we would need to check that as well. 
The release notes of the geopandas version that added the deprecation warning refers to the "geodatasets" tool. Have you had a look at this?

Reply:
I deleted the files and included instructions to download them when necessary in fc5c4a81c91a71b3a24ab43d780ac0e203ff566d and b85dcd77b0af16a1d3658c5236330e5a8146ebe2